test/cfg/test_path_pattern.py

```diff
@@ -64,7 +65,8 @@ def test_mix(mock_devices):
         index=1,
     )
     expected = 'Ext/2023/10/1 + 15-164921'
-    assert str(actual) == expected
+
+    assert str(actual) == os.path.normpath(expected)
```

Comment:
This change is needed to pass the test?

----

I use `pathlib` almost everywhere now instead of `os.path`; the one part that's missing is `os.path.expandvars`: https://docs.python.org/3/library/os.path.html#os.path.expandvars

But I just learned that `os.path.normpath` was slightly different from `Pathlib.resolve`, but now the two are the same.

Since this is partly pedagogical for you, I'm going to suggest you replace tweak these tests as follows:

```
from pathlib import Path

[...]

    assert actual.normalize() == Path(expected).normalize()
```

and remove the `os.path` import. If this were at work, I'd not make that suggestion. :-D

Reply:
I couldn't find anything about the `normalize()` method. I did, however, find an even cleaner way of fixing this.

`Path` provides a nice `match()` method that can be called from its instances. 
https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.match